pyannote/audio/models/separation/ToTaToNet.py

```diff
@@ -137,6 +139,7 @@ def __init__(
         task: Optional[Task] = None,
         n_sources: int = 3,
         use_wavlm: bool = True,
+        wavlm_frozen: bool = False,
```

Comment:
Did you double check that it does not break loading https://huggingface.co/pyannote/separation-ami-1.0?

Reply:
No, I will check it tomorrow.